ts/src/tokocrypto.ts

```diff
@@ -1616,14 +1618,20 @@ export default class tokocrypto extends Exchange {
         //
         if (uppercaseType === 'MARKET') {
```

Comment:
@samgermain we still have some issues here, first if I try to create a market buy using the amount + price, we're not sending the amount*price value in the request, only the amount. Second although the exchange responded with an error we still parse it as if it was a success response

```
✗ n tokocrypto createOrder "ADA/USDT" market buy 1 10 --verbose
Debugger attached.
2023-08-10T14:37:57.234Z
Node.js: v19.5.0
CCXT v4.0.52
tokocrypto.createOrder (ADA/USDT, market, buy, 1, 10)
fetch Request:
 tokocrypto POST https://www.tokocrypto.com/open/v1/orders 
RequestHeaders:
RequestBody:
 timestamp=1691678368441&symbol=ADA_USDT&type=2&side=0&quantity=1&recvWindow=5000&signature=a2383d00b2741ee2184916d8c8a35e31e62edbeae141c8b2a477180216d2514b 

} 
ResponseBody:
 {"code":3210,"msg":"The total volume is too low","data":{"status":"ERROR","type":"GENERAL","code":"989218","errorData":"989218","data":null,"subData":"{\"code\":-1013,\"msg\":\"Filter failure: NOTIONAL\"}","params":["5.00000000","9000000.00000000","{\"code\":-1013,\"msg\":\"Filter failure: NOTIONAL\"}"]},"timestamp":1691678369191} 

2023-08-10T14:39:29.236Z iteration 0 passed in 91533 ms

{
  info: {},
  id: undefined,
  clientOrderId: undefined,
  timestamp: undefined,
  datetime: undefined,
  lastTradeTimestamp: undefined,
  symbol: 'ADA/USDT',
  type: undefined,
  timeInForce: undefined,
  postOnly: false,
  reduceOnly: undefined,
  side: undefined,
  price: undefined,
  stopPrice: undefined,
  triggerPrice: undefined,
  amount: undefined,
  cost: undefined,
  average: undefined,
  filled: 0,
  remaining: undefined,
  status: undefined,
  fee: undefined,
  trades: [],
  fees: [],
  lastUpdateTimestamp: undefined,
  takeProfitPrice: undefined,
  stopLossPrice: undefined
}
2023-08-10T14:39:29.236Z iteration 1 passed in 91533 ms
```

Reply:
I get an error, and also the amount*price value is used

```
2023-08-10T19:16:53.783Z
Node.js: v18.15.0
CCXT v4.0.52
tokocrypto.createOrder (ADA/USDT, market, buy, 1, 0.4)
InvalidOrder tokocrypto {"code":3211,"msg":"The total volume must be greater than 5","timestamp":1691695017880}
---------------------------------------------------
[InvalidOrder] tokocrypto {"code":3211,"msg":"The total volume must be greater than 5","timestamp":1691695017880}

    at throwExactlyMatchedException  …samgermain/Documents/CCXT/ccxt/ts/src/base/Exchange.ts:3426  
    at handleErrors                  …rs/samgermain/Documents/CCXT/ccxt/ts/src/tokocrypto.ts:2454  
    at                               …samgermain/Documents/CCXT/ccxt/ts/src/base/Exchange.ts:1001  
    at processTicksAndRejections     node:internal/process/task_queues:95                          
    at fetch2                        …samgermain/Documents/CCXT/ccxt/ts/src/base/Exchange.ts:2975  
    at request                       …samgermain/Documents/CCXT/ccxt/ts/src/base/Exchange.ts:2979  
    at createOrder                   …rs/samgermain/Documents/CCXT/ccxt/ts/src/tokocrypto.ts:1669  
    at async run                     Users/samgermain/Documents/CCXT/ccxt/examples/ts/cli.ts:301   

tokocrypto {"code":3211,"msg":"The total volume must be greater than 5","timestamp":1691695017880}
```

Reply:
@samgermain that error is expected the minimum amount is 5usd and you're trying to buy 4 (0.4 USD)